src/lib/chips/chip-list.ts

```diff
@@ -184,9 +164,7 @@ export class MatChipList extends _MatChipListMixinBase implements MatFormFieldCo
     return this.multiple ? this._selectionModel.selected : this._selectionModel.selected[0];
   }
 
-  get role(): string|null {
-    return this.empty ? null : 'listbox';
-  }
+  get role(): string | null { return this.empty ? null : 'listbox'; }
```

Comment:
JSdocs for this?

Reply:
"The ARIA role applied to the chip list"